onnx/defs/tensor/defs.cc

```diff
@@ -255,8 +255,11 @@ ONNX_OPERATOR_SET_SCHEMA(
                 return false;
               }
               auto target_elt_type = target_type->tensor_type().elem_type();
+              int64_t saturate = ctx.getAttribute("saturate") != nullptr ? ctx.getAttribute("saturate")->i() : 1;
               FunctionBuilder builder(functionProto);
-              builder.Add("output = Cast (input)", "to", (int64_t)(target_elt_type));
+              builder.Add(
+                  MakeString("output = Cast <to= ", (int64_t)(target_elt_type), ", saturate=", saturate, "> (input)")
```

Comment:
nit: we can just use "saturate: int = @saturate", without explicitly looking up the value of the attribute (in line 258 above).

Reply:
It is meant to be lower-case "saturate" in above, but the @ symbol is reformatting this somehow. Not sure how to escape this special-character

Reply:
done